rma/models/rma_order_line.py

```diff
@@ -108,7 +108,7 @@ def _compute_out_shipment_count(self):
             pickings = line._get_out_pickings()
             line.out_shipment_count = len(pickings)
 
-    def _get_rma_move_qty(self, states, direction="in"):
+    def _get_rma_move_qty(self, states, direction="in", use_demand_qty=True):
```

Comment:
better than adding a signature, do it conditionally based on the state, if the stock move is done use `quantity` if not use `product_uom_qty`